### DISCUSSION TITLE
Customize Defaults

### DISCUSSION THREAD

**Discussion by secretmeister** [naturalcrit/homebrewery#3107] [Q&A] [answered]
I was wondering if it is currently possible (or planned to be possible) to customize the default stat blocks, like when you click spells and has all the default stuff, if we could customize that default. For example, the thing I want to use this for I made a custom magic system, which requires a slightly different format for spell blocks.

**Comment by 5e-Cleric** [comment, marked as answer]
The function that creates the monster statblocks creates a random one by mixing pre-made snippets, and that will likely stay that way.

But, we are working on the idea of every user having their own snippets, we will keep track of the progress of that project [here](https://github.com/naturalcrit/homebrewery/issues/1722).

**Reply by secretmeister** [reply]
Yeah, that sounds perfect! I guess I'll keep an eye on that issue then...

**Comment by calculuschild** [comment]
You can customize very nearly anything in the Homebrewery. It's very common to tweak colors, shapes, sizes, backgrounds, etc., of the different items.  As long as you know some CSS you can style things however you like.

And of course, you can always add or remove sections to the different blocks as you please.

**Reply by 5e-Cleric** [reply]
OP wanted to change the default snippet output.
